homeassistant/components/netgear/config_flow.py

```diff
@@ -129,6 +130,9 @@ async def async_step_ssdp(self, discovery_info: ssdp.SsdpServiceInfo) -> FlowRes
             hostname = cast(str, hostname)
             updated_data[CONF_HOST] = hostname
 
+        if not is_ipv4_address(str(hostname)):
+            return self.async_abort(reason="not_ipv4_address")
```

Comment:
We're missing a test for this. The config flow requires 100% test coverage.

Reply:
Sorry thats my mistake, I will write the tests in a seperate PR.
However I am first going to try to make IPv6 work with the library.